### ISSUE TITLE
Module FastPyRateC is not found on Linux (was working previously)

### ISSUE THREAD

**Issue by bmchorse** [dsilvestro/PyRate#11]
On Linux, FastPyRateC is not found upon using `python PyRate/PyRate.py -v`.

**Version info:**
PyRate v2.0 - 20170811
Linux 2.6.32-696.18.7.e16.x86_64
Python version 2.7.13 (Anaconda distribution)

This is on a high-performance computing cluster. The PyRate folder I am using is up-to-date with master on this repository, and is a submodule of my own project's repository. Previously, using the same project repository and on the same cluster, `FastPyRateC` was found without issues.

In contrast, the same command successfully loads FastPyRateC on my local machine (Windows 10). Again, this is the exact same file structure since the same project repository is synced to my computer.

I will look into the parts of `PyRate.py` where `FastPyRateC` is checked for and loaded to see if I can figure out why it's currently not working but wanted to raise an issue in case anyone else has the chance first.

**Comment by dsilvestro** [comment]
Hey Brianna, 

Did you try to recompile FastPyRateC directly on the cluster? You can follow the instructions in /pyrate_lib/fastPyRateC/README.md

Let me know if that does the trick!
Daniele

**Comment by bmchorse** [comment]
I will try this, thanks!

On Wed, Feb 28, 2018 at 1:04 PM, Daniele Silvestro <notifications@github.com

**Comment by bmchorse** [comment]
Got it working!

When running install.sh, it seems to hang on the 'Unzipping' step. Instead I followed the manual install instructions and when executing `python setup.py build` , I got the following warnings, which do not seem to have ruined the build (pasting here for future reference):

![image](https://user-images.githubusercontent.com/20344778/36913905-664002cc-1e19-11e8-9f49-6e3226d88816.png)

I expected to find the `_FastPyRateC.so` in the ModulePyRate directory, so that threw me off; turns out it was in the `build` subfolder as follows: `./PyRate/pyrate_lib/fastPyRateC/ModulePyRate/build/lib.linux-x86_64-2.7/_FastPyRateC.so`

Once I copied this to the location specified in the instructions (`./PyRate/pyrate_lib/fastPyRateC/Other/`, in this case), overwriting the previous .so file, FastPyRateC was successfully loaded. 

Just wanted to leave this here in case anyone else has troubles finding their newly-created .so file! I imagine the subfolder under `/build/` will be different for other OS's but it gives you somewhere to look.
